xbmc/addons/include/xbmc_pvr_types.h

```diff
@@ -244,6 +244,7 @@ extern "C" {
   {
     char         strGroupName[PVR_ADDON_NAME_STRING_LENGTH]; /*!< @brief (required) name of this channel group */
     bool         bIsRadio;                                   /*!< @brief (required) true if this is a radio channel group, false otherwise. */
+    unsigned int iPosition = 0;                              /*!< @brief (optional) order position of the group */
   } ATTRIBUTE_PACKED PVR_CHANNEL_GROUP;
```

Comment:
Could you please document in the comment that 0 is to be used to indicate that the backend does not support this feature.